swagger/v3/api_spec.json

```diff
@@ -5237,19 +5246,9 @@
             ]
           },
           "schedule_identifier": {
-            "description": "The schedule the participant is enrolled in",
+            "description": "The schedule the participant is enrolled in. For the possible values please refer to the <a href=\"/api-reference/npq/schedules-and-milestone-dates.html#schedules-and-milestone-dates\" class=\"govuk-link\">NPQ schedules and milestone dates guidance</a>.",
```

Comment:
I think because this is an API spec, its safer to have the full enum list on the API spec even if we refer to the guidance, could we please add the full enum list here and everywhere else? (also ECF)